test/forEach.js

```diff
@@ -10,55 +10,9 @@ t.test('forEach', t => {
   t.test('invoke callback for each iteration', t => {
```

Comment:
Do we ever test `forEach`?

Reply:
rather yes than no. We compare sinon.count and store.size, and it tell us that we had iterated through object. but we can add some callback that do someting, do we?